CONTRIBUTING.md

```diff
@@ -55,6 +55,15 @@ On Windows you can also attach from Visual Studio 2017 (Debug | Attach To Proces
 ### Unit Tests
```

Comment:
Can this section be changed to be a `-` list instead of numbered to better show that they are independent? For example, say:

To run unit tests, do one of the following:
- Foo
- Bar
- Baz
- Qux